docs/breez-sdk/src/guide/sdk_building.md

```diff
@@ -0,0 +1,86 @@
+# Building the SDK
```

Comment:
Perhaps it is only me but "Building" is a bit confusing for me. It implies "software building" How about "Customizing the SDK"